views/download.lavender

```diff
@@ -61,7 +61,7 @@ block content
           | Once you have installed and included lavender, the only required
           | configuration is to tell lavender where the views directory is.
           pre
-            code.language-jade.
+            code.language-markup.
```

Comment:
The only thing that was highlighted in this block of code was the first 'Lavender' in the document. I switched it to markup so the Lavender would not be highlighted and it would be less distracting.